src/components/ButtonGroup/__tests__/ButtonGroup.spec.tsx

```diff
@@ -1,33 +1,44 @@
 import React from 'react';
-import toJson from 'enzyme-to-json';
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
 import { ButtonGroup } from '../ButtonGroup';
 import { Button } from '../../Buttons';
 import { Dropdown } from '../../Dropdown';
 import { ListItem } from '../../List/ListItem';
 
 describe('<ButtonGroup> that renders a button', () => {
     it('should render default button correctly', () => {
-        const wrapper = mount(
+        const view = render(
             <ButtonGroup>
                 <Button key="a">A button</Button>
                 <Button key="b">Another button</Button>
             </ButtonGroup>
         );
-        expect(toJson(wrapper)).toMatchSnapshot();
+        expect(view.asFragment()).toMatchSnapshot();
+        const button = screen.getAllByRole('button');
+        expect(button[0]).toHaveClass('ButtonGroup__button--first');
```

Comment:
I think we can apply code style add one empty line.
// Act
// Empty line
// Assert

 expect(view.asFragment()).toMatchSnapshot();
const button = screen.getAllByRole('button');
// empty line
expect(button[0]).toHaveClass('ButtonGroup__button--first');
...

@eszthoff What do you think ?

Reply:
yes, that's a good point. I will change that